src/questions.ts

```diff
@@ -0,0 +1,40 @@
+import * as inquirer from 'inquirer';
+import { basename } from 'path';
+
+const packageQuestions: inquirer.Question[] = [{
+	type: 'input',
+	name: 'package',
+	message: 'What Package to do you want to theme?'
```

Comment:
can you change this to accept multiple package names please as per the POC?

Reply:
Done!